pkg/controller/dataimportcron-controller.go

```diff
@@ -111,7 +111,14 @@ func (r *DataImportCronReconciler) initCron(ctx context.Context, dataImportCron
 			r.log.Error(err, "Unable to create CronJob")
 			return err
 		}
+		if err := r.client.Create(ctx, newInitialJob(cronJob)); err != nil {
```

Comment:
does this resource ever get deleted?

Reply:
see the ttl = 0 added

Reply:
Oh so are jobs currently not getting deleted?

Reply:
In `CronJobSpec` we set both `SuccessfulJobsHistoryLimit` and `FailedJobsHistoryLimit` to 0, so Jobs are deleted at the CronJob level.
But now it's an independent job, so we set `JobSpec.TTLSecondsAfterFinished` to 0. Removing the 2 previous ones since the TTL seems to covers them both doesn't seem important to me.